run_unittests.py

```diff
@@ -317,6 +318,35 @@ def test_string_templates_substitution(self):
         cmd = ['@OUTPUT@.out', 'ordinary', 'strings']
         self.assertRaises(ME, substfunc, cmd, d)
 
+    def test_need_exe_wrapper_override(self):
+        config = ConfigParser()
+        config['binaries'] = {
+          'c': '\'/usr/bin/gcc\'',
```

Comment:
__[flake8]__
* [E121] continuation line under-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->